scapy/supersocket.py

```diff
@@ -116,14 +123,29 @@ def send(self, x):
     if WINDOWS:
         def _recv_raw(self, sock, x):
             # type: (socket.socket, int) -> Tuple[bytes, Any, Optional[float]]
-            """Internal function to receive a Packet"""
+            """Internal function to receive a Packet.
+
+            :param sock: Socket object from which data are received
+            :type sock: socket.socket
```

Comment:
I would argue that `:type:` and `:rtype:` shouldn't be used nowadays, considering we already include those types in type hints.

Sphinx and autodoc already parse the types from the type hints, and modern editors (like VS Code) for instance, also know how to get it. It really is duplicate information, and it's quite ugly IMHO.

Reply:
I agree.

I've removed the type information